benchmarks/FF_LHAPDF_bench.py

```diff
@@ -0,0 +1,149 @@
+"""Benchmark FFs from LHAPDF"""
+
+from banana import register
+
+from eko import interpolation
+from ekomark.benchmark.runner import Runner
+
+register(__file__)
+
+base_operator = {"ev_op_iterations": 50, "backward_inversion": "exact"}
+
+base_theory = {
+    "Qref": 91.1876,
+    "mc": 1.51,
+    "mb": 4.92,
+    "mt": 172.5,
+    "kcThr": 1.0,
+    "kbThr": 1.0,
+    "ktThr": 1.0,
+    "alphas": 0.118000,
+    "alphaqed": 0.007496,
+    "FNS": "ZM-VFNS",
+    "ModEv": "EXA",
+#    "ModEv": "TRN",
+}
+
+FF_sets_lo = [
+    "NNFF10_PIm_lo",
+    "NNFF10_PIp_lo",
+    "NNFF10_PIsum_lo",
+    "NNFF10_KAm_lo",
+    "NNFF10_KAp_lo",
+    "NNFF10_KAsum_lo",
+    "NNFF10_PRm_lo",
+    "NNFF10_PRp_lo",
+    "NNFF10_PRsum_lo",
+]
+FF_sets_nlo = [
+    "NNFF10_PIm_nlo",
+    "NNFF10_PIp_nlo",
+    "NNFF10_PIsum_nlo",
+    "NNFF10_KAm_nlo",
+    "NNFF10_KAp_nlo",
+    "NNFF10_KAsum_nlo",
+    "NNFF10_PRm_nlo",
+    "NNFF10_PRp_nlo",
+    "NNFF10_PRsum_nlo",
+    "MAPFF10NLOPIm",
```

Comment:
we mostly use MAP ones instead of NN where ever possible. However, as far as settings as described on info pages of these FF sets go, they are mostly same.

Reply:
what does "mostly" mean? if they are not the same and you don't want same, maybe drop them? or if they are the exact same then keep them ...

Reply:
well, that depends on what "settings" mean? If you mean settings that specifically concern the benchmark, then I look at nf, xmin, xmax, for example, and  these are the same. However, if you look at Qmax, these are different, but I do not look at them because the Q to which I evolve to is allowed by both. That is why I said "mostly".